.github/workflows/autobuild.yaml

```diff
@@ -29,55 +29,56 @@ jobs:
         shell: cmd
         run: |
           dir
-          
+
       - name: Get commit subject for version checking
         id: get-latest-commit
         run: |
           $latestCommitMessage = git log -1 --pretty=format:%s
-          echo "::set-output name=latest_commit_message::$latestCommitMessage"
-          
+          echo "latest_commit_message=$latestCommitMessage" >> $env:GITHUB_ENV
+
       - name: Extract version from commit message
         id: extract-version
         run: |
-          if ('${{ steps.get-latest-commit.outputs.latest_commit_message }}' -match '^(?<version>[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+)\s-\s') {
+          $commitMessage = $env:latest_commit_message
+          if ($commitMessage -match '^(?<version>[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+)\s-\s') {
             $version = $matches['version']
             Write-Host "Extracted version is: $version"
-            echo "::set-output name=version::$version"
+            echo "version=$version" >> $env:GITHUB_ENV
           } else {
             Write-Host "No valid version found in the latest commit message. Skipping bump."
-            echo "::set-output name=version::"
+            echo "version=" >> $env:GITHUB_ENV
           }
-          
+
       - name: Bump OpenTaiko version if necessary
-        if: steps.extract-version.outputs.version != ''
+        if: env.version != ''
         run: |
-          $newVersion = "${{ steps.extract-version.outputs.version }}"
+          $newVersion = $env:version
           Write-Host "Updating version in OpenTaiko.csproj to $newVersion"
           (Get-Content OpenTaiko/OpenTaiko.csproj) -replace '<Version>.*<\/Version>', "<Version>$newVersion</Version>" | Set-Content OpenTaiko/OpenTaiko.csproj
-          
+
       - name: Get version
         uses: kzrnm/get-net-sdk-project-versions-action@v2
         id: get-version
         with:
           proj-path: OpenTaiko/OpenTaiko.csproj
-          
+
       - name: Get current date
         id: get-date
         run: |
           $date = Get-Date -Format "yyyy-MM-dd"
-          echo "::set-output name=date::$date"
+          echo "date=$date" >> $env:GITHUB_ENV
 
       - name: Get changelogs from commit description
         id: get-commit-message
         run: |
           $message = git log -1 --pretty=%b
-          echo "::set-output name=message::$message"
+          echo "message=$message" >> $env:GITHUB_ENV
 
       - name: Update CHANGELOG.md
         run: |
-          $version = '${{steps.get-version.outputs.version}}'
-          $date = '${{steps.get-date.outputs.date}}'
-          $message = '${{steps.get-commit-message.outputs.message}}'
+          $version = $env:version

```

Comment:
The output of `get-version` was ignored, causing untagged commits fail to be logged into the changelog.